### DISCUSSION TITLE
Feedback on GitHub Mobile App v1.180

### DISCUSSION THREAD

**Discussion by devolution007** [community/community#139804] [Mobile]
I've updated to the latest GitHub mobile app version, but the promised feature to view GitHub activity on user profiles is missing. The 'What's New' description mentions this functionality, but it's not available.

Could you please implement this feature and include the activity widget in user profiles in the next update? Thank you.

Best Regards
[Devolution](https://github.com/devolution007)

**Comment by stevepopovich** [comment]
Hi @devolution007 are you not seeing the new widget, or are you requesting that we also show contributions inside the app in the profile itself?

**Reply by devolution007** [reply]
Yes, I haven't seen the new widget.

**Reply by stevepopovich** [reply]
Did you follow these steps: https://github.blog/changelog/2024-09-17-new-contribution-widget-for-android/

Make sure you are updated and then long press the app -> Widgets

**Reply by devolution007** [reply]
Yes i have followed these steps, but when i select an account after dragging the widget it gives an error : 

![Screenshot_2024-09-27-22-33-49-775_com.miui.home.jpg](https://github.com/user-attachments/assets/c7c37745-adef-49ed-a1e6-0de89c1069dc)

![Screenshot_2024-09-27-22-34-05-293_com.miui.bugreport.jpg](https://github.com/user-attachments/assets/876b1250-16f4-4f42-a85b-8a0dd5f0410e)

**Reply by devolution007** [reply]
> Hi @devolution007 are you not seeing the new widget, or are you requesting that we also show contributions inside the app in the profile itself?

what I'm saying is show contributions inside the app.

**Reply by stevepopovich** [reply]
Do other widgets in other apps work?

**Reply by devolution007** [reply]
> Do other widgets in other apps work?

Yes they are working well, except for the GitHub widget.

**Reply by RyanBarclay** [reply]
@stevepopovich @devolution007 do you think that the contributions graph code for GitHub Mobile might have been broken in latest release?

I am seeing the once working contributions widget on IOS now broken. 
https://github.com/orgs/community/discussions/139998

**Reply by devolution007** [reply]
Its definitely possible, My friend says it was working before the update, but now its not.

**Reply by devolution007** [reply]
Any update on this?

**Reply by stevepopovich** [reply]
At this time, we are not planning to add contributions inside the app, and the widget should be available. We would suggest a reinstall and see if that helps.

**Reply by devolution007** [reply]
I have reinstalled the app, but it didn't work.

**Reply by stevepopovich** [reply]
can you screenshot what you see when you long press the GitHub app please?

**Reply by devolution007** [reply]
https://github.com/user-attachments/assets/cd317c9a-0a7e-4cf8-b197-5eb26f0fa1ae

here you go

**Reply by stevepopovich** [reply]
Ok, great, thank you very much for that. Can you share the device name and operating system version please? You are doing everything correctly, there seems to be compatibility issues with the widget and some operating systems

**Reply by devolution007** [reply]
I'm using Xiaomi Redmi 10 (Model : 21061119AG)

1. MIUI version : MIUI global 14.0.7
2.  Android Version : 13 TP1A.220624.014

**Reply by stevepopovich** [reply]
Ok, thank you. We will need more time to investigate this and will look to resolve this in time. Thank you for all the information!

**Reply by devolution007** [reply]
looking forward to hearing the update from you @stevepopovich 🤗

**Reply by devolution007** [reply]
@stevepopovich Any update regarding widgets?

**Comment by ertugrullyldrm** [comment]
repo #2
